src/common/components/ImageUploader/ImageUploader.js

```diff
@@ -1,36 +1,60 @@
+/* globals cloudmersiveImageApiClient */
+
 import React, { Component } from 'react'
 
 import styles from './ImageUploader.module.scss'
 
+const { REACT_APP_API_KEY } = process.env
+const apiClient = cloudmersiveImageApiClient.ApiClient.instance
+const Apikey = apiClient.authentications['Apikey']
+
 class ImageUploader extends Component {
   constructor (props) {
-    super()
+    super(props)
     this.state = {
-      file: '',
-      imagePreviewURL: ''
+      imagePreviewURL: '',
+      rawFile: ''
     }
 
     this.handleImageChange = this.handleImageChange.bind(this)
     this.handleSubmit = this.handleSubmit.bind(this)
+    this.postImageToApi = this.postImageToApi.bind(this)
+  }
+  postImageToApi () {
+    let currentComponent = this;
```

Comment:
I'm not sure I've seen this pattern before where you make a variable = this, did you read this somewhere or was it  for your own understanding? Genuinely curious.

Reply:
I've used this before, but usually as `let self = this`, in order to scope `this`. But I guess I could change this into an arrow function and accomplish the same thing? 🤔